packages/app/app/components/TrackRow/index.js

```diff
@@ -25,6 +25,10 @@ class TrackRow extends React.Component {
     );
   }
 
+  componentWillMount () {
```

Comment:
`componentWillMount` is deprecated now. Could you move this state initialization to the constructor?
Or better yet, if you want to, you could use `withState` hoc from recompose.